Fix for new update

developer9998 opened the pull request:
-Fixed Forest computer mesh location
-Adjusted Visual and Color/Colour views
-Fixed the group tab (I think)
-More changes but that's all the important ones

developer9998 committed:
minor tweak

-updated credits tab
-possibly fixed names and colours not changing for others

hopefully i didnt fuck this up

developer9998 committed:
fixed name and colour stuff

yeahhhh :///

Graicc committed:
Simplify checks

Graicc committed:
Simplify null checks

developer9998 committed:
fix for newest patch

this should also prevent this issue from happening again

developer9998 committed:
adjusted the keyboard

made the game keys move when you press them just like the ones that the mod adds. probably the worst way to do it but it's still something.

developer9998 committed:
pesky bee

developer9998 committed:
Update ComputerInterface/Views/GameSettings/NameSettingView.cs

Co-authored-by: Graic <33105645+Graicc@users.noreply.github.com>

developer9998 committed:
Update ComputerInterface/Views/GameSettings/NameSettingView.cs

Co-authored-by: Graic <33105645+Graicc@users.noreply.github.com>

developer9998 committed:
Update ComputerInterface/Views/GameSettings/ColorSettingView.cs

Co-authored-by: Graic <33105645+Graicc@users.noreply.github.com>

developer9998 committed:
Update ComputerInterface/Views/GameSettings/ColorSettingView.cs

Co-authored-by: Graic <33105645+Graicc@users.noreply.github.com>

developer9998 committed:
Fix for new update

-Changed the Item view to Visual view
-Edited Visual View
-Edited Colour View
-Fixed Queues
-Fixed Forest computer mesh path
-Added legal disclaimer again
-Extra edits

developer9998 committed:
Merge branch 'main' into main

Graicc commented on ComputerInterface/Views/GameSettings/GroupView.cs:
```diff
@@ -56,11 +57,10 @@ public void DrawHeader(StringBuilder str)
-            str.Append(_selectionHandler.GetIndicatedText(1, "Cave")).AppendLine();
-            str.Append(_selectionHandler.GetIndicatedText(2, "Canyon")).AppendLine();
-            str.Append(_selectionHandler.GetIndicatedText(3, "City")).AppendLine();
-            str.Append(_selectionHandler.GetIndicatedText(4, "Mountain")).AppendLine();
+            str.Append(_selectionHandler.GetIndicatedText(0, "Forest  ")).AppendLine();
+            str.Append(_selectionHandler.GetIndicatedText(1, "Cave    ")).AppendLine();
+            str.Append(_selectionHandler.GetIndicatedText(2, "Canyon  ")).AppendLine();
+            str.Append(_selectionHandler.GetIndicatedText(3, "City    ")).AppendLine();
```
What happened to mountain?

developer9998 replied:
i couldn't get it working.

developer9998 replied:
all of them worked fine but when i tried mountain it didnt work + none worked in mountain

developer9998 replied:
ill prob try to add it back when i have the time..

Graicc replied:
Fixed in commit 7b314ddc3f71260aa748c6729ec6955d1d22870c

Graicc committed:
Clean up code

Graicc committed:
Fix group join index

developer9998 committed:
Limited code and name lengths

this is due to a ban that is going around presumably due to having codes over 12 characters long

developer9998 committed:
Merge branch 'main' of https://github.com/developer9998/ComputerInterface

Graicc committed:
Refactor Group Join

Graicc committed:
Get rid of manual primary color

Graicc committed:
Clamp group join selector

Graicc committed:
Merge branch 'developer9998/main' into HEAD
